src/functionalTest/resources/scenarios/RIA-942-hearing-bundle-is-ready-notification-to-home-office.json

```diff
@@ -0,0 +1,74 @@
+{
+  "description": "RIA-942 Hearing bundle is ready notification to home office",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "CaseOfficer",
+    "input": {
+      "id": 942,
+      "eventId": "generateHearingBundle",
+      "state": "preHearing",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "listCaseHearingCentre": "taylorHouse",
+          "ariaListingReference": "LP/12345/2019",
+          "legalRepReferenceNumber": "REF54321"
+        }
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
+    "errors": [],
+    "caseData": {
+      "template": "minimal-appeal-submitted.json",
+      "replacements": {
+        "listCaseHearingCentre": "taylorHouse",
+        "ariaListingReference": "LP/12345/2019",
+        "legalRepReferenceNumber": "REF54321",
+        "notificationsSent": [
+          {
+            "id": "942_HEARING_BUNDLE_IS_READY_HOME_OFFICE",
+            "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
+          },
+          {
+            "id": "942_HEARING_BUNDLE_IS_READY_LEGAL_REP",
+            "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
+          }
+        ]
+      }
+    },
+    "notifications": [
+      {
+        "reference": "942_HEARING_BUNDLE_IS_READY_HOME_OFFICE",
+        "recipient": "ReformpbCentral@example.com",
```

Comment:
The recipients shouldn't be hard-coded. Should take the following type of format:
"recipient": "{$hearingCentreEmailAddresses.taylorHouse}",